Adyen/Service/BalancePlatform/NetworkTokensService.cs

```diff
@@ -0,0 +1,99 @@
+/*
+* Configuration API
+*
+*
+* The version of the OpenAPI document: 2
```

Comment:
do we need a row on the Readme for Configuration API on this library?
we can have something similar as 

https://github.com/Adyen/adyen-node-api-library/blob/c2e71abdacd90262d5e1310d3e0fbe27a12b6e1f/README.md?plain=1#L22